draft-ietf-rats-corim.md

```diff
@@ -1408,36 +1408,51 @@ Evidence MUST be successfully verified.
 
 ### The Accepted Claims Set
 
-At the end of the Evidence collection process evidence has been converted into
-a format suitable for appraisal. To this end, this document describes an `accepted-claims-set`
-format and the algorithms used to compare it against CoMID Reference Values.
+At the end of the Evidence collection process Evidence has been converted into
+a format suitable for appraisal. To this end, this document describes an
+`accepted-claims-set` format and the algorithms used to compare it against
+CoMID Reference Values.
 
 ~~~ cddl
 {::include cddl/accepted-claims-set.cddl}
 ~~~
 
-Verifiers are not required to use this as their internal state, but for the
+Verifiers are not required to use this as their internal state, for the
 purposes of this document a sample Verifier is discussed which uses this format.
 
-The Accepted Claims Set will be matched against CoMID Reference Values, as per
+The Accepted Claims Set (ACS) contains the actual state of Target Environments (TEs).
+The `evidence-triples` field contains Evidence (from Attesters) and Endorsements
+(e.g. from `endorsed-triple-record`).
+
+CoMID Reference Values will be matched against the Accepted Claims Set, as per
 the appraisal policy of the Verifier.
 This document describes an example evidence structure which can be easily
 matched against these Reference Values.
 
-Each set of evidence contains an `environment-map` providing a namespace, and
-a non empty `measurement-values-map`.
+Each entry within `evidence-triples` uses the syntax of `endorsed-triple-record`.
+When an `endorsed-triple-record` appears within `evidence-triples` it
+indicates that the authority named by `measurement-map`/`authorized-by`
+asserts that the actual state of the Target Environment `environment-map`
+has the values in `measurement-map`/`mval`.
+
+In `authorized-by`, authority is represented by cryptographic keys. Authority
+is asserted by digitally signing a Claim using the key. Hence, Claims are
+added to the ACS under the authority of a key.
 
-Each entry in the `measurement-values-map` is a separate piece of evidence
+Each entry in the `measurement-values-map` is Evidence or an Endorsement
```

Comment:
Please create an issue as to how Endorsement matching instead of Evidence is relevant here.

Reply:
Can you file the issue? I am not sure what you would like to see changed.